Task/Taskfile.yml

```diff
@@ -89,7 +89,7 @@ tasks:
       vars: ['VERSION']
     vars:
       VERSION: '{{.VERSION}}'
-      PLATFORM: '{{.PLATFORM | default .LOCAL_PLATFORM}}'
+      PLATFORM: '{{if eq .PLATFORM "all"}}{{.SUPPORTED_PLATFORMS}}{{else if .PLATFORM}}{{.PLATFORM}}{{else}}{{.LOCAL_PLATFORM}}{{end}}'
```

Comment:
This only needs to be done here and not on `publish` because publish will pass along a var that's set, or default to `LOCAL_PLATFORM` if unset like we expect in this logic.  If all is set, it just gets passed and then interpreted / overwritten here